test/driver/ocaml-ppx-context-load-path-migration/dune

```diff
@@ -0,0 +1,13 @@
+; Remember to bump the enabled if to the latest supported OCaml
```

Comment:
It's been a while so I need to dive back into this but I think the idea is that before 5.02 there were no changes in the format of `ocaml.ppx.context` so if we can round trip from the latest supported version to 5.1 and back it should be enough. That way there is no need to make this test compatible with a range of compiler versions, just the latest.

Reply:
Yes. I'm just missing why it's not automatically compatible with all compiler versions >=5.02. If it's compatible, I'd prefer to remove the remarks about it having to be the latest and just having a >=5.02 constraint in the dune file to avoid the extra maintenance burden of having to bump that version manually once in a while. If it's not automatically compatible with all >=5.02 versions, I agree that just having those remarks and manually bumping the version from time to time is better/less effort than making it cross-version-compatible on purpose.

Reply:
In my opinion the maintenance burden would be to maintain this test for all versions rather than just one (the latest).

I'll remove the comment and we'll decide what to do when 5.3 comes up.